src/composables/useTour.ts

```diff
@@ -0,0 +1,393 @@
+                    '.account-overview .bitcoin-account',
+                ],
+                disabledNextStep: true,
+            },
+        },
+        [MobileOnboardingTourStep.ACCOUNT_OPTIONS]: {
+            path: '/?sidebar=true',
+            tooltip: {
+                target: '.modal .small-page',
+                content: [
+                    'Create, switch and log out of accounts.',
+                    'All security relevant actions can be found here too.',
+                ],
+                params: {
+                    placement: 'top',
+                },
+            },
+            ui: {
+                elementsWithoutInteractivity: [
+                    '.column-sidebar',
+                ],
+            },
+            lifecycle: {
+                prepareDOMPrevPage: async () => {
+                    await closeAccountOptionsModal();
+
+                    root.$router.push('/');
+                    await sleep(500); // TODO Check this random value
+                },
+                prepareDOMNextPage: async () => {
+                    await closeAccountOptionsModal();
+                },
+            },
+        },
+        [MobileOnboardingTourStep.ONBOARDING_COMPLETED]: {
+            path: '/?sidebar=true',
+            tooltip: {
+                target: '.column-sidebar .network .consensus-icon',
+                content: [
+                    'Wallet tour completed!',
+                    'Nimiq is not just any crypto - Click on {WORLD} Network and discover true decentralization.',
+                ],
+                params: {
+                    placement: 'top-start',
+                },
+            },
+            ui: {
+                elementsWithoutInteractivity: [
+                    '.column-sidebar',
+                ],
+            },
+            lifecycle: {
+                prepareDOMPrevPage: async () => {
+                    await openAccountOptionsModal();
+                },
+            },
+        },
+    };
+    steps[MobileOnboardingTourStep.TRANSACTIONS_LIST].lifecycle = {
+        ...steps[MobileOnboardingTourStep.TRANSACTIONS_LIST].lifecycle,
+
+        onMountedStep: () => {
+            root.$watch(() => useTransactionsStore().state.transactions, (txs) => {
+                if (Object.values(txs).length > 0) {
+                    // Once the user has at least one transaction, tooltip in step TRANSACTIONS_LIST is modified
+                    steps[MobileOnboardingTourStep.TRANSACTIONS_LIST].tooltip = {
+                        target: '.vue-recycle-scroller__item-wrapper',
+                        content: ['This is where all your transactions will appear.'],
+                        params: {
+                            placement: 'bottom',
+                        },
+                    };
+                    steps[MobileOnboardingTourStep.TRANSACTIONS_LIST].ui.disabledNextStep = false;
+                }
+            });
+        },
+    };
```

Comment:
Why is this outside/after the above step definition?

Reply:
I thought that you couldn't reference variable `step` inside of its own declaration like: 

```
const step = {
   fn: () => console.log(step)
}
```

But I was wrong. You can't do that when you are declaring a property of a class but here is not a problem